x-pack/plugins/security_solution/kibana.json

```diff
@@ -18,6 +18,7 @@
     "eventLog",
     "features",
     "inspector",
+    "lens",
```

Comment:
We have lens as an option plugin also, can we add dependency for the only single place

Reply:
Yes, I've removed the optionalPlugins and keep it in requiredPlugins